ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/tools/osb/mbean/ObjStoreBrowser.java

```diff
@@ -80,14 +80,20 @@
     private static OSBTypeHandler[] defaultJTSOsbTypes = {
             new OSBTypeHandler(
                     true,
-                    false,
                     "com.arjuna.ats.internal.jta.tools.osb.mbean.jts.JTSXAResourceRecordWrapper",
                     "com.arjuna.ats.internal.jta.tools.osb.mbean.jts.JTSXAResourceRecordWrapper",
                     "CosTransactions/XAResourceRecord",
                     null
             ),
             new OSBTypeHandler(
                     true,
+                    "com.arjuna.ats.internal.jta.tools.osb.mbean.jts.JCAServerTransactionWrapper",
```

Comment:
I made the change to be consistent with the existing AA code in that you have to enable it directly now.

I agree with your suggestion that the docs just need to say to suspend the automated recovery manager and control it by hand. - please can you raise something in the docs?